study-builder/fdahpStudyDesigner/src/main/java/com/fdahpstudydesigner/controller/CleanPropertyAdvice.java

```diff
@@ -1,6 +1,7 @@
 package com.fdahpstudydesigner.controller;
 
 import java.beans.PropertyEditorSupport;
+import org.apache.commons.text.StringEscapeUtils;
```

Comment:
add file header with license and copyright like other .java files

Reply:
Added header and copyright to the modified file